docs/src/SUMMARY.md

```diff
@@ -97,9 +98,16 @@
 - [ADR-007: restructuring](./adr/007adr-restructuring.md)
 - [ADR-008: exceptions](./adr/008adr-exceptions.md)
 - [ADR-009: outputs](./adr/009adr-outputs.md)
+- [RFC-010: Implementation of Transition Exploration Server](./adr/010rfc-transition-explorer.md)
```

Comment:
I don't think this should be linked into the manual. It's a purely internal technical doc and doesn't add value to our user documentation IMO.